src/decanter/core/client.py

```diff
@@ -48,9 +48,7 @@ def __init__(self, username, password, host):
         
 
     @staticmethod
-    def setup(
-            train_data, data_source, data_columns, data_id=None, callback=None,
-            eda=None, preprocessing=None, version=None, name=None):
+    def setup(setup_input, name=None):
         """Setup data reference.
 
         Create a DataUpload Job and scheduled the execution in CORO_TASKS list.
```

Comment:
Need to update all document part to fit deata setup

Reply:
done